composer.json

```diff
@@ -4,6 +4,7 @@
     "license": "MIT",
     "require": {
         "php": ">7.1.0,<7.3.0",
+        "jetbrains/phpstorm-stubs": "dev-master",
```

Comment:
Obviously, this cannot stay as-is :-(

Reply:
Yes. I will ask them again. Or we could fork the repository and make releases...

Reply:
I asked the team about tagging releases [on Twitter](https://twitter.com/ntzm_/status/969205170319093762), and it was added to their issue tracker [here](https://youtrack.jetbrains.com/issue/WI-40958).

Reply:
Nice, thanks @ntzm ! :+1:

Reply:
@ntzm Perfect, it looks promising :)

Reply:
https://github.com/JetBrains/phpstorm-stubs/releases/tag/2018.1.2